### ISSUE TITLE
[BUG] - Messages remain unread in Axolotl

### ISSUE THREAD

**Issue by AA303** [nanu-c/axolotl#172]
### Description

Messages remain unread in Axolotl even after they are read in the desktop client. 

### Steps to Reproduce

1. As a sender to send you a message and read it on the desktop client

**Expected behavior:** [What you expect to happen]

The message should be marked as read in Axolotl chat list.

**Actual behavior:** [What actually happens]

The message remains unread in Axolotl, until you open the chat.

#### Versions
0.8.7
Please provide the Version as written in `Settings->Advanced-> Submit Debug Log`

#### Device
Nexus 5

#### Link to Debug Log
Please provide a link to debug

**Comment by nanu-c** [comment]
The ticks in axolotl only show:
1. tick, successful sent to signal server.
2. tick the signal server acknowledge the message.
That's only triggered, when sending messages from axolotl. I could lie and always but 2 ticks if they are sent from signal desktop. 
Or properly implement message read messages...

**Comment by nanu-c** [comment]
duplicate of #91 and it's going to be solved

**Comment by AA303** [comment]
I have upgraded to 0.9.0 the issue is not resolved. When I read entire conversation on linked Desktop Client - the Axolotl still displays unread counter on the list of conversations.

**Comment by Flaburgan** [comment]
Yes, this is indeed not related to ticks and is still a problem, I'm reopening.
